backtrace-library/src/main/java/backtraceio/library/breadcrumbs/BacktraceBreadcrumbsLogManager.java

```diff
@@ -13,7 +13,7 @@
 
     private final String LOG_TAG = BacktraceBreadcrumbsLogManager.class.getSimpleName();
 
-    private long breadcrumbId = 0;
+    private long breadcrumbId = System.currentTimeMillis();
```

Comment:
I think we have a problem with unit test project right now because of this change. my recommendation:
* you can expose breadcrmbId and start using it in unit tests so we will know start number,
* we can move breadcrumbs changes to separated pull requesst

Reply:
Yes, we don't want any changes to Breadcrumbs as side effect.

Reply:
Thank you for sharing how to fix UTs by exposing breadcrumb ID, I will make this change!